app/models/piece.js

```diff
@@ -36,7 +38,7 @@ export default class Piece extends Model {
   translationActivityGeneratedBy;
   @belongsTo('sign-marking-activity', { inverse: 'piece', async: true })
   signMarkingActivity;
-  @belongsTo('signed-piece', { inverse: 'unsignedPiece', async: true }) signedPiece;
+  // @belongsTo('signed-piece', { inverse: 'unsignedPiece', async: true }) signedPiece;
```

Comment:
Deze mag integraal weg ten voordele van L20-21 denk ik. As an aside: Toelichten van eventuele afwijkingen van gedocumenteerd model doen we meestal in resource config.